### PR TITLE
Fixes list of valid cron expressions

### PR DESCRIPTION
**Pull request by swadeley**
cron in Linux does not support the use of seconds #6633

### CODE REVIEW

**Comment by swadeley** [comment]
robottelo]$ pytest  tests/foreman/cli/test_syncplan.py::SyncPlanTestCase::test_positive_update_description   
========================== test session starts =========================
platform linux -- Python 3.6.6, pytest-4.0.2, py-1.6.0, pluggy-0.8.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/sjw/Downloads/robottelo, inifile:
plugins: xdist-1.23.2, services-1.3.1, mock-1.10.0, forked-0.2, cov-2.6.0
collecting ... 2019-01-10 21:43:57 - conftest - DEBUG - BZ deselect is disabled in settings

collected 1 item                                                                                                             

tests/foreman/cli/test_syncplan.py .                                                                                   [100%]

====================== 1 passed in 43.91 seconds ========================

**Comment by swadeley** [comment]
"man crontab.5" tells us what should work

File: robottelo/datafactory.py [hunk 1]
```diff
@@ -684,11 +684,11 @@ def valid_cron_expressions():
     """Returns a list of valid cron expressions"""
     return [
         # After 30 min
-        "0 */30 * * *",
+        "*/30 * * * *",
         # At noon every day
-        "0 0 12 * * ",
+        "0 12 * * * ",
```
Comment (ldjebran) [inline robottelo/datafactory.py]: Please remove the space eg: "0 12 * * *"

**Comment by swadeley** [comment]
@ldjebran thank you for the review,

File: robottelo/datafactory.py [hunk 2]
```diff
@@ -684,11 +684,11 @@ def valid_cron_expressions():
     """Returns a list of valid cron expressions"""
     return [
         # After 30 min
-        "0 */30 * * *",
+        "*/30 * * * *",
         # At noon every day
-        "0 0 12 * * ",
+        "0 12 * * *",
         # Weekdays at 2:00am
```
Comment (omkarkhatavkar) [inline robottelo/datafactory.py]: change this comment to 
"At 02:00 AM, Monday through Friday"

Comment (omkarkhatavkar) [reply robottelo/datafactory.py]: No need I think, ignore above comment

**Pull request merged**